epimodel/exports/epidemics_org.py

```diff
@@ -532,6 +555,7 @@ def process_export(args) -> None:
             get_df_list(timezone_df, code),
             get_df_else_none(un_age_dist_df, m49),
             get_df_else_none(traces_v3_df, iso3),
+            get_df_else_none(countermeasures_df, code),
```

Comment:
I believe you can do `get_df_list` to ensure that you do not have to do L175-L177.

Reply:
I got an error when I tried to use `get_df_list`. But I'll look into it.

Reply:
If you post which one, then we can help.